app/abtests/Test.scala

```diff
@@ -0,0 +1,54 @@
+package abtests
+
+import com.github.slugify._
+import play.api.libs.json.{JsValue, Json, Writes}
+import play.api.mvc.Cookie
+
+case class Test(name: String, audienceSize: Double, audienceOffset: Double, variants: Variant*) {
```

Comment:
I think it only makes sense if the maximum test ID is 100, in which case you can think of an `Int` size and offset as a percentage. Otherwise it probably makes more sense (or is at least more general) to have them as `Doubles`.